src/main/java/xbot/common/command/BaseRobot.java

```diff
@@ -117,6 +91,22 @@ protected BaseComponent getInjectorComponent() {
      * This function is run when the robot is first started up and should be used for any initialization code.
      */
     public void robotInit() {
+
+        Logger.getInstance().recordMetadata("ProjectName", "XbotProject"); // Set a metadata value
+        if (isReal() || forceWebots) {
+            Logger.getInstance().addDataReceiver(new WPILOGWriter("/media/sda1/")); // Log to a USB stick
```

Comment:
do we need to guard against the stick being missing or is this pretty tolerant?

Reply:
From my memory last year the robot wouldn't crash; it would just fail to log. But should be revisited.